src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -732,6 +734,54 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     ),
                     default_result_type="duration",
                 ),
+                fields.MetricsFunction(
+                    "http_error_count",
+                    snql_metric_layer=lambda args, alias: Function(
+                        "http_error_count",
+                        [
+                            self.resolve_mri(
+                                "span.self_time"
+                            ),  # gets stripped by the mqb transformer
+                            resolve(
+                                UseCaseID.SPANS,
+                                self.builder.organization_id,
+                                self.resolve_mri("span.self_time").name,
+                            ),
+                            resolve_tag_key(
+                                UseCaseID.SPANS,
+                                self.builder.organization_id,
+                                "span.status_code",
+                            ),
```

Comment:
I thought we didn't want to expose these resolve functions outside of the metrics layer?

Reply:
That's a fair point. I might have to go back to the drawing board.